web-service-consumer/1.5/modules/ROOT/pages/web-service-consumer-consume.adoc

```diff
@@ -91,6 +91,9 @@ The optional `headers` element contains application-specific information (like a
 </wsc:consume>
 ----
 
+Note that inside the DW script the root must be `"headers"`. Otherwise, the following error returns while running the application `Invalid input headers XML: It must be an xml with the root tag named \'headers\'`.
```

Comment:
```suggestion
Note that inside the DW script, the root must be `"headers"`, otherwise, the following error returns while running the application: 

`Invalid input headers XML: It must be an xml with the root tag named \'headers\'`.
```